subprojects/docs/src/docs/userguide/javaPlugin.xml

```diff
@@ -1378,6 +1378,15 @@
             </para>
         </section>
 
+        <section id="junit_custom_runners">
+            <title>JUnit custom runners</title>
+            <para>To instantiate a custom runner JUnit provides abstract Request class.</para>
```

Comment:
@diacone You said this well at https://github.com/gradle/gradle/pull/701#issuecomment-251334155

Please consider something like "A JUnit Custom Runner can be configured to select a runner without annotating test classes or to wrap already customized test classes with another runner."

"To configure a Custom Runner, implement ... as shown in this example:"